services/metadactyl-clj/src/metadactyl/persistence/app_metadata/relabel.clj

```diff
@@ -20,9 +17,9 @@
   [app-id]
   (let [tasks (get-tasks-for-app app-id)]
     (when (not= 1 (count tasks))
-      (throw+ {:error_code ce/ERR_ILLEGAL_ARGUMENT
-               :reason     :NOT_SINGLE_STEP_APP
-               :step_count (count tasks)}))
+      (throw+ {:type :clojure-commons.exception/illegal-argument
+               :error     :NOT_SINGLE_STEP_APP
+               ::step_count (count tasks)}))
```

Comment:
Why the double-colon here? It didn't seem to be used for any of the other extra keys above. Similarly, why a keyword for the error rather than a string as used elsewhere above?